test/e2e/autoscaling/custom_metrics_stackdriver_autoscaling.go

```diff
@@ -101,13 +165,69 @@ var _ = SIGDescribe("[HPA] Horizontal pod autoscaling (scale resource: Custom Me
 			},
 		}
 		metricTargets := map[string]int64{"metric1": metric1Target, "metric2": metric2Target}
-		deployment := monitoring.StackdriverExporterDeployment(stackdriverExporterDeployment, f.Namespace.ObjectMeta.Name, int32(initialReplicas), containers)
-		customMetricTest(f, f.ClientSet, podsHPA(f.Namespace.ObjectMeta.Name, stackdriverExporterDeployment, metricTargets), deployment, nil, initialReplicas, scaledReplicas)
+		tc := CustomMetricTestCase{
+			framework:       f,
+			kubeClient:      f.ClientSet,
+			initialReplicas: initialReplicas,
+			scaledReplicas:  3,
+			// Metric exported by deployment is ignored
```

Comment:
Where's the metric value coming from? There's no exporter pod except for deployment in this test case.

Reply:
The comment was a lie, we use the metric from deployment. Deleted.